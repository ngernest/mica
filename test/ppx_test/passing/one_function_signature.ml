module type S = sig
  type 'a t

  val f : 'a t -> 'a t
end
[@@deriving_inline mica_types]

type expr = F of expr
type ty = IntT

[@@@end]
