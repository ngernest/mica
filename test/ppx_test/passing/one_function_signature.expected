module type S  = sig type 'a t val f : 'a t -> 'a t end[@@deriving_inline
                                                         mica_types]
type expr =
  | F of expr 
type ty =
  | IntT 
let rec gen_expr ty = failwith "TODO"
[@@@end ]
